internal/boxcli/midcobra/telemetry.go

```diff
@@ -144,11 +102,19 @@ func (m *telemetryMiddleware) newEventIfValid(cmd *cobra.Command, args []string,
 		return nil
 	}
 
-	pkgs := getPackages(cmd)
+	pkgs, hash := getPackagesAndCommitHash(cmd)
 
 	// an empty userID means that we do not have a github username saved
 	userID := telemetry.UserIDFromGithubUsername()
 
+	devboxEnv := map[string]interface{}{}
+	for _, e := range os.Environ() {
+		if strings.HasPrefix(e, "DEVBOX") && strings.Contains(e, "=") {
+			key := strings.Split(e, "=")[0]
+			devboxEnv[key] = os.Getenv(key)
```

Comment:
instead of another call to Getenv, why not do:
```
parts := strings.Split(e, "=")
key = parts[0]
devboxEnv[key] = parts[1]
```
?

Reply:
Because I actually don't know if `Split()` will return an array of size 2. it may return a larger array if the value of the variable happens to have a `=` in it...
It's not expensive to call `Getenv()` (I think), so it seems safer to just call it